django/cantusdb_project/main_app/views/chant.py

```diff
@@ -89,7 +85,7 @@ def parse_json_from_ci_api(path: str) -> Optional[list]:
     assert path[0] != "/"
     url: str = f"https://cantusindex.org/{path}"
     try:
-        response: Optional[Response] = requests.get(
+        response: Optional[Response] = get(
```

Comment:
You should not de-namespace this -- it's important to know *what* get you're calling here, particularly in a request handler that, itself, defines it's own `get` method.